flattentool/tests/test_input_SpreadsheetInput.py

```diff
@@ -531,4 +531,32 @@ def __str__(self):
         == "2015-06-01T13:37:59+01:00"
     )
 
-    assert len(recwarn) == 0
+    # Should this be "wkt" as the "type" instead of "geojson"?
```

Comment:
Personally I'd split this into multiple tests, this test is doing lots of things that don't need to be related and that may cause annoyances while working on convert_type. However as that is how it already was, I'm ok with leaving it.

Reply:
I've split the new tests out.